src/Chainweb/BlockHeader.hs

```diff
@@ -23,6 +23,7 @@
 {-# LANGUAGE TypeInType #-}
 {-# LANGUAGE TypeOperators #-}
 {-# LANGUAGE UndecidableInstances #-}
+{-# LANGUAGE ViewPatterns #-}
```

Comment:
Where is this used?

Reply:
I don't recall but something was complaining that it was missing or I wouldn't have added it